docs/userguide/envvars.md

```diff
@@ -0,0 +1,110 @@
+Darmstadt, Germany and/or its affiliates. The recording of metrics is turned off for
+all other users and impossible due to a VPN block. In any case, the usage statistics
+do **not** involve logging of recorded measurements, targets or any project information
+that would allow for reconstruction of details. The user and host machine names are
+irreversibly anonymized.
+```
+
+Monitored quantities:
+* `batch_size` used when querying recommendations
+* Number of parameters in the search space
+* Number of constraints in the search space
+* How often [`recommend`](baybe.campaign.Campaign.recommend) was called
+* How often [`add_measurements`](baybe.campaign.Campaign.add_measurements) was called
+* How often a search space is newly created
+* How often initial measurements are added before recommendations were calculated
+  ("naked initial measurements")
+* The fraction of measurements added that correspond to previous recommendations
+* Each measurement is associated with an irreversible hash of the user- and hostname
+
+The following environment variables control the behavior of BayBE telemetry:
+- `BAYBE_TELEMETRY_ENABLED`: Flag that can turn off telemetry entirely (default is
+  `True`). To turn it off set it to `False`.
+- `BAYBE_TELEMETRY_ENDPOINT`: The receiving endpoint URL for telemetry data.
+- `BAYBE_TELEMETRY_VPN_CHECK`: Flag turning an initial telemetry connectivity check
+  on/off (default is `True`).
+- `BAYBE_TELEMETRY_VPN_CHECK_TIMEOUT`: The timeout in seconds for the check whether the
+  endpoint URL is reachable.
+- `BAYBE_TELEMETRY_USERNAME`: The name of the user executing BayBE code. Defaults to an
+  irreversible hash of the username according to the OS.
+- `BAYBE_TELEMETRY_HOSTNAME`: The name of the machine executing BayBE code. Defaults to
+  an irreversible hash of the machine name.
+
+```{admonition} Uninstalling Internet Packages
+:class: important
+If you do not trust the instructions above, you are free to uninstall all
+internet-related packages such as `requests` or `opentelemetry*`. These are being
+shipped in the default dependencies because there is no good way of creating opt-out
+dependencies, but the baybe package will work without them.
+```
+
+## Caching
+For some components, such as the
+[`SubstanceParameter`](baybe.parameters.substance.SubstanceParameter), some of the
+computation results are cached both in memory and in local storage.
+
+By default, BayBE determines the location of temp files on your system and puts cached
+data into a subfolder `.baybe_cache` there. If you want to change the location of the
+disk cache, change:
+```bash
+BAYBE_CACHE_DIR="/path/to/your/desired/cache/folder"
+```
+
+By setting
+```bash
+BAYBE_CACHE_DIR=""
+```
+you can turn off disk caching entirely.
+
+## Floating Point Precision
+In general, double precision is recommended as numerical stability during optimization
+can be bad when single precision is used. This impacts gradient based optimization,
```

Comment:
```suggestion
In general, double precision is recommended because numerical stability during optimization
can be bad when single precision is used. This impacts gradient-based optimization,
```